File: JanperCab/5-JanperCab/ClientApp/src/app/_models/enquiry/EnquiryForSheetDto.ts
export class EnquiryForSheetDto {
  enquiryId: number;
  customerId: number;
  customerName: string;
  customerReference: string;
  doorType: string;
  doorColor: string;
  partCount: number;
  address: string;
  suburb: string;
  state: string;
  postcode: string;

  getFullAddress = (): string => {
    return `${this.address}, ${this.suburb} ${this.state} ${this.postcode}`;
  };
}

File: JanperCab/3-Application/Dtos/Enquiry/EnquiryForSheetDto.cs
namespace _3_Application.Dtos.Enquiry
{
    public class EnquiryForSheetDto
    {
        public int EnquiryId { get; set; }

        public int CustomerId { get; set; }

        public string CustomerName { get; set; }

        public string CustomerReference { get; set; }

        public string DoorType { get; set; }

        public string DoorColor { get; set; }

        public int PartCount { get; set; }

        public string Address { get; set; }

        public string Suburb { get; set; }

        public string State { get; set; }

        public string Postcode { get; set; }

        public EnquiryForSheetDto(_1_Domain.Enquiry enquiry)
        {
            EnquiryId = enquiry.Id;
            CustomerId = enquiry.CustomerId;
            CustomerName = enquiry.Customer.Name;
            CustomerReference = enquiry.CustomerReference;
            DoorType = enquiry.DoorType;
            DoorColor = enquiry.DoorColor;
            PartCount = enquiry.PartCount;
            Address = enquiry.DeliveryAddress;
            Suburb = enquiry.DeliverySuburb;
            State = enquiry.DeliveryState;
            Postcode = enquiry.DeliveryPostcode;
        }
    }
}